ihmc-footstep-planning/src/main/java/us/ihmc/footstepPlanning/graphSearch/stepChecking/HeightMapFootstepChecker.java

```diff
@@ -189,13 +188,6 @@ private void doValidityCheck(DiscreteFootstep candidateStep, DiscreteFootstep st
 
    private BipedalFootstepPlannerNodeRejectionReason doValidityCheckForTerrainMap(DiscreteFootstep candidateStep)
    {
-      double areaFraction = candidateStepSnapData.getSnapAreaFraction();
-      if (Double.isFinite(areaFraction))
-         footAreaPercentage.set(areaFraction);
-      else
-         footAreaPercentage.set(1.0);
-      rmsError.set(candidateStepSnapData.getSnapRMSError());
```

Comment:
This is already contained in the snap, and is inside the doValidityCheckForSnap method.